trillian/ctfe/cache/cache.go

```diff
@@ -15,7 +15,21 @@
 // Package cache defines the IssuanceChainCache type, which allows different cache implementation with Get and Set operations.
 package cache
 
-import "context"
+import (
+	"context"
+	"errors"
+	"flag"
+	"time"
+
+	"github.com/google/certificate-transparency-go/trillian/ctfe/cache/lru"
+	"github.com/google/certificate-transparency-go/trillian/ctfe/cache/noop"
+)
+
+var (
+	cacheType = flag.String("cache_type", "noop", "Supported cache type: noop, lru (Default: noop)")
+	size      = flag.Int("cache_size", -1, "Size parameter set to 0 makes cache of unlimited size")
+	ttl       = flag.Duration("cache_ttl", -1*time.Second, "Providing 0 TTL turns expiring off")
+)
```

Comment:
It's best practice (or at least traditional/standard) to define flags up at the top level `main` file. Is there are a reason to break precedent in this project and define the flags down here?

Reply:
Moved cache flags to ct_server `main`.